sale_global_discount/models/sale_order.py

```diff
@@ -115,6 +115,8 @@ def _compute_amounts(self):
                         1.0,
                         product=line.product_id,
                         partner=line.order_id.partner_shipping_id,
+                        # By default, handle_price_include=True. In case of apply discount
+                        handle_price_include=False,
```

Comment:
so this will be false regardless of `line.tax_id.price_include`?

Reply:
would we need to impact `_convert_to_tax_base_line_dict` as well in account_tax.py?

Reply:
> so this will be false regardless of `line.tax_id.price_include`?

Hi anh, If I understand correctly, yes, value of param does not effect the base amount in case tax is not included in price:
https://github.com/odoo/odoo/blob/d53869c203ecd9e9eccb95b958d77cc5c23ebe59/addons/account/models/account_tax.py#L613

Reply:
> would we need to impact `_convert_to_tax_base_line_dict` as well in account_tax.py?

Yes, I think it's needed, given that it does not change behavior when tax_id.price_include=False